### PR TITLE
Dump

### PR DESCRIPTION
**Pull request by bariscaglar**
Resolved a compiler warning by removing the Platform attribute from Package. The use of Platform attribute is discouraged.

Also renamed the readme file because the GitHub page was trying to show it, failing in the process.

### CODE REVIEW

File: Installer/iiscompression.wxs [hunk 1]
```diff
@@ -139,7 +138,7 @@
               <File Id="ReadmeFile"
                     ShortName="readme.htm"
                     Name="iiscompression_readme.htm"
-                    Source=".\readme.htm"
+                    Source=".\iiscompression_readme.htm"
```
Comment (bangbingsyb) [inline Installer/iiscompression.wxs]: How doe GitHub page fail to show the original readme.htm? It seems that I can see its content from master branch. Why does renaming it solve the issue?

File: Installer/Setup.wixproj [hunk 2]
```diff
@@ -13,9 +13,11 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Platform)' == 'x86' ">
     <OutputName>iiscompression_x86</OutputName>
+    <InstallerPlatform>x86</InstallerPlatform>
```
Comment (bangbingsyb) [inline Installer/Setup.wixproj]: What is purpose of InstallerPlatform?

Comment (bariscaglar) [reply Installer/Setup.wixproj]: This gets passed on to Package in place of Platform, whose use is discouraged.

File: Installer/iiscompression.wxs (hunk 1 above)
Comment (bariscaglar) [reply Installer/iiscompression.wxs]: Github page, when you navigate to that folder, finds a readme.htm and tries to display it, but it is only able to display md files properly. When you rename it, it is not found by GitHub so when you navigate to that directory, it doesn't try to display it. This is only for the source file; the name of the readme file that gets installed on the target computer is unchanged.

Comment (bangbingsyb) [reply Installer/iiscompression.wxs]: Ah, I see. It is the folder page that shows the contents of readme.html - a weird GitHub convention.

**Pull request merged**